### PR TITLE
Cucumber 3 changes

### PR DESCRIPTION
**Pull request by mikhaillopandia**
Changes have been applied according to https://cucumber.io/blog/2017/09/21/upgrading-to-cucumber-3

### CODE REVIEW

File: features/support/transformers.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#############################################################
+#                      TRANSFORMERS                         #
+#############################################################
+
+ParameterType(
+  name: 'page',
+  regexp: /(.+)/,
+  transformer: lambda do |page|
```
Comment (alexey-voronenko) [inline features/support/transformers.rb]: For one line transformers it's better to use functional style:
```
 transformer: -> (page) { page }
```

File: features/support/transformers.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    page
+  end
+)
+
+ParameterType(
+  name: 'menu item',
+  regexp: /(.+?)/,
+  transformer: lambda do |menu_intem|
```
Comment (alexey-voronenko) [inline features/support/transformers.rb]: same here

File: features/support/transformers.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    menu_intem
+  end
+)
+
+ParameterType(
+  name: 'email',
+  regexp: /(.+)/,
+  transformer: lambda do |email|
```
Comment (alexey-voronenko) [inline features/support/transformers.rb]: same

File: features/support/transformers.rb [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    email
+  end
+)
+
+ParameterType(
+  name: 'employment type',
+  regexp: /(.+)/,
+  transformer: lambda do |email|
```
Comment (alexey-voronenko) [inline features/support/transformers.rb]: same

**Review by alexey-voronenko — CHANGES_REQUESTED** [review]
See comments above

**Comment by mikhaillopandia** [comment]
@Lec1k thank you a lot for review, changes have been applied, check now, please.

**Review by romikoops — APPROVED** [review]

**Pull request merged**